sapi_manifests/madtom_coal/template

```diff
@@ -192,29 +150,10 @@
       "processType": "redis",
       "checkerType": "redis"
     },
-    {
-      "processType": "stud",
-      "checkerType": "http",
-      "path": "/ping",
-      "secure": true,
-      "rejectUnauthorized": {{MANTA_REJECT_UNAUTHORIZED}}
-    },
-    {
-      "processType": "supervisor",
-      "checkerType": "http",
-      "path": "/kang/snapshot",
-      "secure": false
-    },
     {
       "processType": "workflow",
```

Comment:
"workflow" was a mythical Manta service that never existed. If you like you could drop "workflow" entries as well.

Reply:
Yes, dropped.